src/app/[locale]/specs/xrpc/en.mdx

```diff
@@ -187,6 +187,12 @@ Only HTTPS should be used over the open internet.
 
 Care should be taken with personally identifiable information in blobs, such as EXIF metadata. It is currently the *client's* responsibility to strip any sensitive EXIF metadata from blobs before uploading. It would be reasonable for a PDS to help prevent accidental metadata leakage as well; see future changes section below.
 
+## Redirects
+
+XRPC is built on top of HTTP. As such, everything that is not explicitly allowed or forbidden by the XRPC spec falls under the authority of HTTP.
+
+This means that all *un-authenticated* "Query" (HTTP GET) XRPC endpoints, such as `getBlob`, support redirects.
```

Comment:
Unauthenticated "Query" (HTTP GET) XRPC endpoints, such as `getBlob`, support redirects.

An explicit decision has not been made around XRPC support for redirects for "Procedure" (HTTP POST) endpoints and therefore should not be relied on.

HTTP clients should take care to strip `Authorization` headers for cross-origin redirects. Most browsers and stdlib HTTP clients handle this behavior

Reply:
Hm -- I don't think adding "generally" here would be clarifying in that case, and I agree that the framing here is a bit more sensitive in a spec than it would be in a guide, but I also think this is a very good thing to reinforce in cases like this. Is there a rewrite that still expresses this that you think would fit here?